### ISSUE TITLE
Add apps shortcuts to the desktop

### ISSUE THREAD

**Issue by iamafasha** [anbox/anbox#1704]
**Please describe your problem:**
Mine is not a bug, but I tried to search for the solution and am not seeing it. How can I add a shortcut to just a specific app I installed on Anbox? Is it possible?

**Comment by firemeteorxx** [comment]
Isn't it automatic?
I didn't do anything manually but the following show up in my case:

~/.local/share/applications/anbox
$ cat anbox-com-android-calculator2.desktop
[Desktop Entry]
Name=Calculator
Exec=/usr/bin/anbox launch --action=android.intent.action.MAIN --package=com.android.calculator2 --component=com.android.calculator2.Calculator 
Terminal=false
Type=Application
Icon=/home/firemeteor/.local/share/applications/anbox/anbox-com-android-calculator2.png

You can do it manually if it does not show up for any reason.

**Comment by miezebieze** [comment]
Like hinted by firemeteorxx Anbox creates a .desktop file for each app in ~/.local/share/applications/anbox which makes them show up in your launcher/start menu.

It depends on your desktop environment how you get them to show up on your desktop.

**Comment by iamafasha** [comment]
> Like hinted by firemeteorxx Anbox creates a .desktop file for each app in ~/.local/share/applications/anbox which makes them show up in your launcher/start menu.
> 
> It depends on your desktop environment how you get them to show up on your desktop.

Am using Linux mint 20 and there us no folder `anbox` in `~/.local/share/applications/ `  
Here is how it looks like
![image](https://user-images.githubusercontent.com/43771428/102374536-827f6080-3fd2-11eb-8b51-c6e0d8a08741.png)

**Comment by steelcowboy** [comment]
I'm on Ubuntu 20.10 and after a fresh install `~/.local/share/applications/anbox` seems to be missing for me as well

**Comment by Nishant0073** [comment]
My solution  to launch WhatsApp is as follows:

I have created on script to launch WhatsApp :

```
#!/bin/bash 
URI=`adb shell pm list packages | grep whatsapp`
ACTIVITY=`adb shell cmd package resolve-activity --brief ${URI#package:} | tail -n 1`
anbox launch --package=${URI#package:} --component=${ACTIVITY//$'/'/}
```

And then one desktop launcher is as follows:

[Desktop Entry]
Version=1.0
Type=Application
Terminal=false
Exec= bash path_to_script.sh
Name=WhatsApp
Comment=anbox WhatsApp
Icon=whatsapp.svg

**Comment by iamafasha** [comment]
> I'm on Ubuntu 20.10 and after a fresh install `~/.local/share/applications/anbox` seems to be missing for me as well
> 
> **EDIT**: This seems to be expected, see #749. For a permanent solution, you can add the following to `/etc/fstab`, replacing <YOUR_USERNAME> with your username:
> 
> ```
> /home/<YOUR_USERNAME>/snap/anbox/common/app-data/applications/anbox
> /home/<YOUR_USERNAME>/.local/share/applications/anbox none defaults,bind 0 0
> ```
> 
> Then do:
> 
> ```
> mkdir -p ~/.local/share/applications/anbox
> sudo mount ~/.local/share/applications/anbox
> ```

```
iamafasha@iamafasha-HP-ProBook-450-G6:~$ sudo mount ~/.local/share/applications/anbox
mount: /etc/fstab: parse error at line 13 -- ignored
mount: none: mount point does not exist.

```
I get this error when you try this

**Comment by joshm24923** [comment]
@iamafasha , try modifying the fstab so the two directories are part of the same line of code like this. I think the example @steelcowboy  provided may have had a return in it:
`/home/<YOUR_USERNAME>/snap/anbox/common/app-data/applications/anbox /home/<YOUR_USERNAME>/.local/share/applications/anbox none defaults,bind 0 0`

**Comment by steelcowboy** [comment]
@iamafasha @joshm24923  yes, I tried to remove the return but GitHub wouldn't let it be a single wrapped line :(

**Comment by joshm24923** [comment]
Single vs triple tilde for code blocks :facepalm: . Same mistake on my side.

```
/home/<YOUR_USERNAME>/snap/anbox/common/app-data/applications/anbox /home/<YOUR_USERNAME>/.local/share/applications/anbox none defaults,bind 0 0
```

**Comment by erik9012** [comment]
I didnt have shortcuts too on my linux mint. What I had to do was enabling the permission for each .desktop file in the directory home/MYNAME/local/share/applications/anbox to run as program and then  to make it shortcuts i copied them to my homescreen. I would like it if anyone can explain what happens here above me whit thing like the sintence" you can add the following to /etc/fstab" because i think in that case i get automated shortcuts i think

**Comment by HukuToc2288** [comment]
Editing fstab seems to be overkill. I just created symbolic link
`ln -s ~/snap/anbox/common/app-data/applications/anbox ~/.local/share/applications/anbox`
and it worked perfectly

**Comment by Fuseteam** [comment]
Yeah this should be fixed
